packages/manager/modules/pci/src/projects/project/serving/delete/translations/Messages_fr_FR.json

```
@@ -2,7 +2,7 @@
   "pci_projects_project_serving_delete": "Suppression du namespace",
   "pci_projects_project_serving_delete_common_delete": "Supprimer",
   "pci_projects_project_serving_delete_common_cancel": "Annuler",
-  "pci_projects_project_serving_delete_warning": "Attention ! Vous vous apprêtez à supprimer votre namespace. L'ensemble des modèles et tokens associés seront égalements supprimés.",
+  "pci_projects_project_serving_delete_warning": "Attention ! Vous vous apprêtez à supprimer votre namespace. L’ensemble des modèles et tokens associés seront également effacés.",
```

```suggestion
  "pci_projects_project_serving_delete_warning": "Attention ! Vous vous apprêtez à supprimer votre namespace. L’ensemble des modèles et tokens associés sera également effacé.",
```

`L’ensemble` will be removed.

---

Good idea to suggest French corrections in English